# horizontal
float64 phi_min
float64 phi_inc
uint32  phi_n

# vertical
float64 theta_min
float64 theta_inc
uint32  theta_n

# range
float64 range_min
float64 range_inc
uint32  range_n

# additional radar parameters
float64 wavelength
float64 deviation
